examples/demos/example14-all-features-immutable.js

```diff
@@ -189,6 +189,8 @@ export default class extends React.Component {
           onGridRowsUpdated={this.handleGridRowsUpdated}
           rowHeight={50}
           minHeight={600}
+          enableCellCopyPaste
```

Comment:
Should we remove this whole example as we don't support immutable any more?

Reply:
I think we should revisit demos and remove obsolete/duplicate examples. Probably suited for a separate PR